modules/skparagraph/src/TextWrapper.cpp

```diff
@@ -122,13 +122,24 @@ void TextWrapper::lookAhead(SkScalar maxWidth, Cluster* endOfClusters) {
                 }
                 // If the word is too long we can break it right now and hope it's enough
                 fMinIntrinsicWidth = std::max(fMinIntrinsicWidth, nextWordLength);
-                if (fClusters.endPos() - fClusters.startPos() > 1 ||
+                // By default, the words that are wider than the available width are wrapped in such
```

Comment:
It probably doesn't matter since I guess you can tell with the git diff, but would be able to denote that this comment is something we've added?

Reply:
I think you can easily do it with Git blame

Reply:
Do you regularly use git blame when looking at our skia checkout? My concern is about the visibility of this being a behavioural change we've added to skia, not whether we can eventually figure out whether we added it or not.

Reply:
What are you suggesting we should do?

Reply:
I think just adding that to the top of the comment. Something such as "Non-skia upstreamed behavioural change."

Reply:
Done 👍